jhon_esteban/tarea_1/html/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/jhon_esteban/tarea_1/css/style.css">
+    <title>Magazine Cover</title>
+</head>
+<body>
+
+    <div class="container">
+        <div class="top">
+            <div class="name">
+                <h1>Katya </h1>
+                <h1>Zamolodchikova</h1>
+                <h3>Dragqueen</h3>
+            </div>
+            
+            <div class="season">
+                <h1>Rupaul's</h1>
+                <h1>Drag Race</h1>
+                <h1>AllStars</h1>
+                <h1> S2 top 4</h1>
+            </div>
+        </div>  
+        <div class="center">
+            <div>
+                <h1>Your Majesty</h1>
+                <h1>Miss </h1>
+                <h1>Congeniality</h1>
+                <h3>She was achosen as a favorite <h3>by fans and contestants</h3> <h3>for her absurd sense of humor</h3></h3>
+            </div>
+            <div class="voltiao">
+                <h1>Listen now!</h1>
+                <h1>Read u wrote U</h1>
+            </div>
+        </div>
+        <div class="bottom">
+            <div>
+                <h1>Youtube Channel Trixie & Katya</h1>
+                <h5>August 2022</h5>
+                <h5>welovekatya.com</h5>
+            </div>
+            <div>
+                <div class="makeup">
+                    <img src="/jhon_esteban/tarea_1/img/runways.webp" alt="">
```

Comment:
@Jhonec0814 , you should use relative paths 

```suggestion
                    <img src="img/runways.webp" alt="">
```